styles/mixins.scss

```diff
@@ -0,0 +1,17 @@
+@mixin font-details($color, $family, $size, $transform) {
```

Comment:
Am sentimente *mixte* legate de mixinul ăsta. Chiar ai nevoie de toți 4 parametri mereu?

Măcar `$transform` l-aș face opțional, dacă nu și la altele. Sass are blocuri de [flow control](https://sass-lang.com/documentation/at-rules/control), inclusiv `@if`

```scss
@if $transform {
    text-transform: $transform;
}
```